Makefile

```diff
@@ -25,15 +25,15 @@ test: clean
 
 .PHONY: test-unit
 test-unit: clean
-	mvn test -Dgroups="uk.gov.companieshouse.extensions.api.groups.Unit"
+	mvn test -Dgroups=unit
```

Comment:
on master branch: 'make test-unit' runs successfully
on this branch: 'make test-unit' is failing for me.

Reply:
I find it is passing - do you have the lastest code locally?

Reply:
i believe so, it is attempting to connect to mongo then timing out. Ian's works though.

Reply:
my test-unit works, but test-integration is attempting to connect to a db and timing out.

Reply:
Run this on master and find that this is the same:
RequestRepositioryIntegrationTest – all – DataAccessResourceFailiureException right?
I don't think these work locally.

Reply:
on master branch, 'make test-unit' doesn't run ContractProviderIntegrationTest.
But on this branch, 'make test-unit' does run ContractProviderIntegrationTest.
You can replicate this by adding a System.out in the ContractProviderIntegrationTest.setup to identify when it runs.
It was originally failing for me as i had a pact folder with pacts that were being executed when i ran 'make test-unit'. The System.out in the setup method will help you identify when the test is being executed.